install/shake.yaml

```diff
@@ -1,5 +1,5 @@
 # Used to provide a different environment for the shake build script
-resolver: lts-14.11 # GHC 8.6.5
+resolver: lts-14.17 # GHC 8.6.5
```

Comment:
I think we should leave bumping resolvers for the monthly release process, for older GHC versions.

Reply:
Reverted.
I had bumped, to and `stack-8.6.5.yaml`, so stack does not have to download multiple snapshots.